package/upgrade/migrations/managed_charts/logging-event-audit-v1.0.3.yaml

```diff
@@ -0,0 +1,104 @@
+# This file is only used in v1.0.3 -> v1.1.0 upgrade path
+# Please keep syncing with harvester, harvester-installer v1.1.0 version
+#   harvester/harvester-installer/pkg/config/templates/rancherd-14-logging-crd.yaml
+#   harvester/harvester-installer/pkg/config/templates/rancherd-15-logging.yaml
+resources:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: cattle-logging-system
+- apiVersion: management.cattle.io/v3
+  kind: ManagedChart
+  metadata:
+    name: rancher-logging-crd
+    namespace: fleet-local
+  spec:
+    chart: rancher-logging-crd
+    releaseName: rancher-logging-crd
+    version: "100.1.3+up3.17.7"
+    defaultNamespace: cattle-logging-system
+    repoName: harvester-charts
+    targets:
+    - clusterName: local
+      clusterSelector:
+        matchExpressions:
+        - key: provisioning.cattle.io/unmanaged-system-agent
+          operator: DoesNotExist
+- apiVersion: management.cattle.io/v3
+  kind: ManagedChart
+  metadata:
+    name: rancher-logging
+    namespace: fleet-local
+  spec:
+    chart: rancher-logging
+    releaseName: rancher-logging
+    version: "100.1.3+up3.17.7"
```

Comment:
Wouldn't this version be missing the patches to `rancher-logging` made in the installer?

https://github.com/harvester/harvester-installer/tree/c33e9647e6b9567e695c98d7c45392498aaf874c/pkg/config/templates/patch/rancher-logging/100.1.3%2Bup3.17.7

Reply:
The patch is processed in buding ISO, the `harvester-repo` has upgraded chart file.  

When upgrading, the `rancher-logging` chart points to the patched one.

```
hack/patch-rancher-logging:  echo "patch rancher logging rke2 configmap.yaml, daemonset.yaml"
hack/patch-rancher-logging:  cp -f ${pkg_logging_path}/${logging_version}/configmap.yaml ./rancher-logging/templates/loggings/rke2/configmap.yaml
hack/patch-rancher-logging:  grep "harvester-patch" ./rancher-logging/templates/loggings/rke2/configmap.yaml || true

build-bundle:source ${SCRIPTS_DIR}/hack/patch-rancher-logging
```